maven/pom-core.xml

```diff
@@ -12,6 +12,32 @@
   <description>Thin Java abstractions for the standard built-in objects for Javascript.</description>
   <url>https://www.gwtproject.org</url>
 
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <connection>scm:git:git@github.com:google/elemental2.git</connection>
+    <developerConnection>scm:git:git@github.com:google/elemental2.git</developerConnection>
+    <url>git@github.com:google/elemental2.git</url>
```

Comment:
This should link to the GitHub repo home: https://github.com/google/elemental2 (that element is supposed to denote the "publicly browsable repository": https://maven.apache.org/pom.html#SCM)

Reply:
Good point - fixed